diff --git a/node_modules/snowpack/lib/build/optimize.js b/node_modules/snowpack/lib/build/optimize.js
index c070f78..8e7d733 100644
--- a/node_modules/snowpack/lib/build/optimize.js
+++ b/node_modules/snowpack/lib/build/optimize.js
@@ -417,7 +417,7 @@ async function runBuiltInOptimize(config) {
         for (const bundledInput of Object.keys(manifest.inputs)) {
             if (!manifest.outputs[bundledInput]) {
                 logger_1.logger.debug(`Removing bundled source file: ${path_1.default.resolve(buildDirectoryLoc, bundledInput)}`);
-                await fs_1.promises.unlink(path_1.default.resolve(buildDirectoryLoc, bundledInput));
+                await fs_1.promises.unlink(path_1.default.resolve(originalCwd, bundledInput));
             }
         }
         rimraf_1.default.sync(path_1.default.resolve(buildDirectoryLoc, util_1.removeLeadingSlash(config.buildOptions.webModulesUrl)));
